docs/DafnyRef/Topics.md

```diff
@@ -561,7 +565,7 @@ such that $k \leq \ell$:
  {\;{}^{\flat}\kern-1mm f}_k    \quad\;\dot{\Rightarrow}\;\quad {\;{}^{\flat}\kern-1mm f}_\ell \quad\;\dot{\Rightarrow}\;\quad f      \quad\;\dot{\Rightarrow}\;\quad {\;{}^{\sharp}\kern-1mm f}_\ell \quad\;\dot{\Rightarrow}\;\quad { {}^{\sharp}\kern-1mm f}_k $$</p>
 
 In other words, every ${\;{}^{\flat}\kern-1mm f}\_{k}$ is a _pre-fixpoint_ of $f$ and every ${\;{}^{\sharp}\kern-1mm f}\_{k}$ is a _post-fixpoint_
```

Comment:
There appears to be a bunch of stray `\;` and `\:` commands that aren't being rendered correctly as spacing, in this section and later ones too.

Reply:
Again -- these are rendering OK with the GitHub preview. How are you viewing it?

Reply:
Seems that the GitHub "rich diff" view doesn't render them correctly, but the preview you linked does. Thanks.